### PR TITLE
LogsVolumePanel: remove unneccessary filters and parsers in query expression

### PR DESCRIPTION
**Pull request by gtk-grafana**
Extends the fix in https://github.com/grafana/explore-logs/pull/628 to work for logs volume panel.

### CODE REVIEW

File: src/Components/ServiceScene/Breakdowns/LabelBreakdownScene.tsx [hunk 1]
```diff
@@ -349,33 +340,6 @@ export class LabelBreakdownScene extends SceneObjectBase<LabelBreakdownSceneStat
...
-    }
-    const streamSelectors = [...labelsVariable.state.filters, labelExpressionToAdd]
-      .filter(isDefined)
-      .map((f) => `${f.key}${f.operator}\`${f.value}\``)
-      .join(',');
-
-    const fields = fieldsVariable.state.filters;
-    const levels = levelsVariables.state.filters;
-    // if we have fields, we also need to add `VAR_LOGS_FORMAT_EXPR`
-    if (fields.length || levels.length) {
```
Comment (gtk-grafana) [inline src/Components/ServiceScene/Breakdowns/LabelBreakdownScene.tsx:373]: oopsie I broke Matias's latest changes here

Comment (matyax) [reply src/Components/ServiceScene/Breakdowns/LabelBreakdownScene.tsx:373]: :(

**Review by matyax — APPROVED** [review]
Looks great to me!

**Comment by gtk-grafana** [comment]
After merging in main I'm seeing bug on this branch with the log volume histogram showing no-data. :(

**Pull request merged**